src/modules/feedback/components/mapFeedback/MapFeedbackPanel.js

```diff
@@ -0,0 +1,192 @@
+import { html } from 'lit-html';
+import { $injector } from '../../../../injection';
+import { MvuElement } from '../../../MvuElement';
+import css from './mapFeedbackPanel.css';
+
+const Update_Category = 'update_category';
+const Update_Description = 'update_description';
+const Update_EMail = 'update_email';
+const Update_CategoryOptions = 'update_categoryoptions';
+const Update_Geometry_Error_Display = 'update_geometry_error_display';
+const Update_Geometry_Valid = 'update_geometry_valid';
+
+export class MapFeedbackPanel extends MvuElement {
+	constructor() {
+		super({
+			mapFeedback: {
+				state: '',
+				category: '',
+				description: '',
+				email: '',
+				fileId: ''
```

Comment:
This field holds the id of a created geometry. It Should be initially `null`...
If it's `null` when the user submits the "Geometry is missing" message should be displayed.